source/slang/slang-ir-dominators.cpp

```diff
@@ -427,6 +439,7 @@ struct DominatorTreeComputationContext
     {
         // First we compute the postorder traversal order for the blocks in the CFG.
         computePostorder(code, postorder);
+        computeReachableSet(code, reachableSet);
```

Comment:
Can this be fused with `computePostorder`, so that we only need to traverse the graph once?